OWNERS

```diff
@@ -3,8 +3,10 @@ approvers:
 - NickrenREN
 - dims
 - FengyunPan2
+- gonzolino
```

Comment:
Let's start as a reviewer and then we can move to approver after some time. is that ok?

Reply:
Yes of course